tot/experiment.py

```diff
@@ -72,6 +74,10 @@ def __post_init__(self):
                 "experiment": self.experiment_name,
             }
 
+        scaler = self.params.pop("scaler", None)
```

Comment:
@ankke It would be cleaner to not pass the scaler with the model_params. However, I would suggest leaving it for now and refactoring once redoing the param handling